tactics/tacticals.ml

```diff
@@ -434,9 +434,13 @@ module New = struct
     tclIFTHENELSE t1 (tclTRY t2) t2
 
   (* Try the first tactic that does not fail in a list of tactics *)
-  let rec tclFIRST = function
-    | [] -> tclZEROMSG (str"No applicable tactic.")
-    |  t::rest -> tclORELSE0 t (tclFIRST rest)
+  let rec tclFIRST ?(msg = str "No applicable tactic.") = function
+    | [] -> tclZEROMSG msg
+    | t::rest -> tclORELSE0 t (tclFIRST ~msg rest)
+
+  let rec tclMULTIFIRST ?(msg = str "No applicable tactic.") = function
```

Comment:
Actually no, this is not the same tactical. `tclANY` takes only one tactic and a list of arguments. 
- Do we have a similar tactical without backtracking so that the code of `eassumption` can be as factorized as possible?
- Would it make sense to move `tclANY` from `tactics.ml` (where it is not exported) to `tacticals.ml`?